pkg/api/server/db/model_test.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/tektoncd/results/pkg/api/server/db/errors"
+	"github.com/tektoncd/results/pkg/api/server/test"
+)
+
+func TestQueryAnnotation(t *testing.T) {
```

Comment:
Nice thinking with the test! It might be simpler just to test the Scan/Value functions directly though (e.g. similar to our tests of ToAPI/ToStorage) - as is this basically a simplified version of the results tests below.

Reply:
Actually this is different from results test because in the results test, every query is wrapped by gorm, the receiver of gorm query is always the db model, so we can't test  the nil annotation pointer behavior:
```go
db.Select("name", "age").Find(&users)
```

However, anyway, I think just to test the Scan/Value functions is a great idea! And this part is done!